git/README.md

```diff
@@ -63,10 +63,13 @@ We use widely established conventions for idiomatic Git commit messages:
 ## Releases process
 
 In order to release new version of a software: 
- - Create a new branch for each soon to be released version. For example if we are going to release `release/0.21`, we should create such branch.
- - Create a pre-release with `0.21.0-rc0` tag and start testing it.
- - When team finishes testing, release should be made from that `release/0.21` branch with a tag `0.21.0`.
- - All future critical bugs affecting network operation should be backported to `release/0.21` branch for 30 days since its initial release.
+ 1) Decide if a release branch is required. (i.e. you should create release branch if there are commits in master branch you don't want to release)
+    - If required: create branch reflecting release version. If release version is `0.21` then branch name must be `release/0.21`.
+    - If not: continue release from master
+ 2) Create a **pre-release** in GitHub and tag appropriate branch (see 1.) with tag named `0.21.0-rc1`.
+ 3) After pipeline completes and publishes release artifacts make an internal announcement in slack to start testing pre-release.   
```

Comment:
This can be automated via CI.

Reply:
this is actually good idea, I will create a task

Reply:
https://github.com/mysteriumnetwork/node/issues/3299

Reply:
Release branch was suggested because for minors releases (e.g. 0.21.1) we should simply cherry picking one or two commits to that `release/0.21` branch and releasing from there.

Reply:
It's common to have a `dev` branch and keep that slight divergence between master with that. Then tagging can be done on a commit at `dev` and once released it can be merged into master and dev goes on. Have you considered this approach @chompomonim @soffokl @mdomasevicius ?

Reply:
We have considered many approaches. But also team changes a lot. We have new team members, some people left... I think we can have any new rules you guys agree here.